test/integ/file_upload.cpp

```diff
@@ -470,6 +513,43 @@ LT_BEGIN_AUTO_TEST(file_upload_suite, file_upload_memory_only_incl_content)
     delete ws;
 LT_END_AUTO_TEST(file_upload_memory_only_incl_content)
 
+LT_BEGIN_AUTO_TEST(file_upload_suite, file_upload_large_content)
```

Comment:
I think we might also benefit from a test that combines this with arguments being passed with the POST